src/BookBundle/Controller/UserController.php

```diff
@@ -151,4 +158,26 @@ public function newAdminAction(Request $request)
             'form' => $form->createView(),
         ));
     }
+
+    /**
+     *
+     * @Route("/one-index", name="one_wilder_index")
+     * @return \Symfony\Component\HttpFoundation\Response
+     * @Security("has_role('ROLE_USER')")
+     */
+    public function indexWilderAction()
+    {
+        $userId = $this->getUser()->getId();
+
+        $em = $this->getDoctrine()->getManager();
+        $wild = $em->getRepository('BookBundle:Wilder')->getWilderUser($userId);
```

Comment:
ce n'est pas nécessaire non ? 
ne peux tu pas faire un findByUser($this->getUser()) directement ? ou je ne comprends pas bien ce que tu veux faire sinon...